src/nwbinspector/checks/nwbfile_metadata.py

```diff
@@ -36,6 +36,20 @@ def check_session_start_time_old_date(nwbfile: NWBFile):
         )
 
 
+@register_check(importance=Importance.BEST_PRACTICE_SUGGESTION, neurodata_type=NWBFile)
+def check_session_start_time_contains_time_zone(nwbfile: NWBFile):
+    """
+    Check if the session_start_time contains a time zone.
+
+    Best Practice: :ref:`best_practice_global_time_reference`
+    """
+    session_start_time = nwbfile.session_start_time
+    if session_start_time.tzinfo is None:
```

Comment:
```suggestion
    if isinstance(session_start_time, datetime) and session_start_time.tzinfo is None:
```

Reply:
@CodyCBakerPhD , does this address your comment below?

Reply:
It does - specifically, it seems `datetime.date` does not even have the `tzinfo` attribute otherwise

Can you add a test for that skip case?